services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/ProjectSettingsManagerImpl.java

```diff
@@ -111,19 +139,32 @@ public ProjectSetting createProjectSetting(UserInfo userInfo, ProjectSetting pro
 
 		// make sure the project id is a project
 		EntityType nodeType = nodeManager.getNodeType(userInfo, parentId);
```

Comment:
If of the certified type then add the extra check but all other logic still applies.